cli/cmd/repair.go

```diff
@@ -3,17 +3,25 @@ package cmd
 import (
 	"errors"
 	"fmt"
+	"regexp"
 	"time"
 
 	"github.com/golang/protobuf/ptypes"
+	"github.com/linkerd/linkerd2/controller/api/public"
 	pb "github.com/linkerd/linkerd2/controller/gen/config"
 	"github.com/linkerd/linkerd2/pkg/charts/linkerd2"
+	"github.com/linkerd/linkerd2/pkg/healthcheck"
 	"github.com/linkerd/linkerd2/pkg/k8s"
+	"github.com/linkerd/linkerd2/pkg/version"
 	"github.com/spf13/cobra"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/yaml"
 )
 
+var (
+	versionRegex = regexp.MustCompile("stable-2.[8-9].*")
```

Comment:
Why considering `stable-2.8.*`? I think this should only apply to `stable-2.9.*`.

Reply:
Because there is no repair cmd in `2.8` and if the cmd suggests then prob they might except that sub cmd to exist. WDYT?

Reply:
Okay, That makes sense. Updated to fail and suggest upgrading for versions below `2.9`